doc/PACKS.md

```diff
@@ -55,6 +55,7 @@ Currently supported flags:
 * `"exlorom"`: (SNES) game has ExLoROM mapping
 * `"exhirom"`: (SNES) game has ExHiROM mapping
 * `"sa-1"`: (SNES) game has SA-1 mapping
+* `"luaconnector"`:(Bizhawk) LuaConnector de crowd control
```

Comment:
hm, what is "de" ?

Maybe this?
```suggestion
* `"luaconnector"`: Connect to (Bizhawk) LuaConnector (crowd control version)
```

Reply:
mhh sorry I think in French and normally I write in English, but in this case I put a ‘de’ instead of ‘of’.